os.go

```diff
@@ -0,0 +1,22 @@
+package tableroll
+
+import "os"
+
+type osIface interface {
+	Getpid() int
+	FindProcess(pid int) (processIface, error)
```

Comment:
What about making these not exported? They are just for testing, right?

Reply:
I'd rather leave it as identical to the `os` package since this is a interface for that. I think it's clearer that it's a mock that by default is identical to `os.Getpid` by leaving the capitalization.

Reply:
Fine with me.